{"uid":"59a223c0c6ec5622","name":"Проверить обновление счетчика просмотров проекта","fullName":"tests.test_profile.TestProfile#test_update_views_counter","historyId":"5d2f140f5151a80fc8ff5958afb6560a","time":{"start":1738865306211,"stop":1738865306211,"duration":0},"description":"Проверить, что счетчик изменяется при входе и открытии страницы","descriptionHtml":"<p>Проверить, что счетчик изменяется при входе и открытии страницы</p>\n","status":"broken","statusMessage":"KeyError: 'access_token'","statusTrace":"request = <SubRequest 'login_to_app' for <Function test_update_views_counter>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"77fa487050b552e11569bdc395ea9724\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n>       login_page.user(User.LOGIN, User.PASSWORD)\n\nconftest.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.api.authorization.ApiAuthorization object at 0x7f0ee2e25340>\nusername = 'Aleska', password = 'qwe123'\n\n    def user(self, username: str, password: str) -> None:\n        \"\"\"\n        Метод авторизации пользователя по средствам API.\n    \n        :param username: Логин пользователя\n        :param password: Пароль пользователя\n        \"\"\"\n        headers = {\n            'X-Client-Id': 'round',\n            'X-Client-Type': 'web',\n            'X-Language': 'ru'\n        }\n        data = {\n            'client_id': 'Krujok.Online.Mobile',\n            'grant_type': 'password',\n            'username': username,\n            'password': password\n        }\n    \n        response = requests.post(url=Api.TOKEN, headers=headers, data=data)\n>       access_token = response.json()[\"access_token\"]\nE       KeyError: 'access_token'\n\nmodel/api/authorization.py:30: KeyError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1738864340794,"stop":1738864340795,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1738865306212,"stop":1738865306628,"duration":416},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1738865306628,"stop":1738865306629,"duration":1},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1738865306628,"stop":1738865306628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_update_views_counter>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"setup","time":{"start":1738865306628,"stop":1738865306628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Авторизация пользователя в системе","time":{"start":1738865306629,"stop":1738865307129,"duration":500},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[{"name":"Авторизация пользователя в системе","time":{"start":1738865306629,"stop":1738865307128,"duration":499},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/conftest.py\", line 53, in login_to_app\n    login_page.user(User.LOGIN, User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'login_to_app' for <Function test_update_views_counter>>"},{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"77fa487050b552e11569bdc395ea9724\")>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1738865307135,"stop":1738865307311,"duration":176},"status":"passed","steps":[],"attachments":[{"uid":"d79a88bdf2c4d148","name":"page_source","source":"d79a88bdf2c4d148.html","type":"text/html","size":39},{"uid":"4057d1aaa599b37c","name":"screenshot","source":"4057d1aaa599b37c.png","type":"image/png","size":10592},{"uid":"9c003e86acb7d2b0","name":"browser_logs","source":"9c003e86acb7d2b0.txt","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"feature","value":"Профиль пользователя"},{"name":"severity","value":"Minor"},{"name":"tag","value":"regression"},{"name":"tag","value":"profile"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_profile"},{"name":"subSuite","value":"TestProfile"},{"name":"host","value":"6af98febc332"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_profile"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"minor","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":10,"skipped":0,"passed":30,"unknown":0,"total":43},"items":[{"uid":"dac6907a31b37b63","status":"passed","time":{"start":1738269272700,"stop":1738269278270,"duration":5570}},{"uid":"267e92ee5307668","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1738099704426,"stop":1738099704426,"duration":0}},{"uid":"51131d9db6fce30d","status":"passed","time":{"start":1738005594556,"stop":1738005601451,"duration":6895}},{"uid":"dd6db7511fb91167","status":"passed","time":{"start":1738002884325,"stop":1738002889585,"duration":5260}},{"uid":"a461b8aa35532b25","status":"broken","statusDetails":"urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=52493): Read timed out. (read timeout=120)","time":{"start":1737931756493,"stop":1737931756493,"duration":0}},{"uid":"c2cadf972dee4ff9","status":"passed","time":{"start":1737843252183,"stop":1737843259279,"duration":7096}},{"uid":"21a4148567208229","status":"passed","time":{"start":1737832446824,"stop":1737832453593,"duration":6769}},{"uid":"eaf651ecea736239","status":"passed","time":{"start":1737799808918,"stop":1737799815795,"duration":6877}},{"uid":"23caa916b1d0ff4d","status":"passed","time":{"start":1737758039227,"stop":1737758046092,"duration":6865}},{"uid":"fc9a971d00269483","status":"passed","time":{"start":1737745036065,"stop":1737745042937,"duration":6872}},{"uid":"a1c30b4a24574ea9","status":"passed","time":{"start":1737652921527,"stop":1737652930019,"duration":8492}},{"uid":"97714366cfba8d08","status":"passed","time":{"start":1737570600385,"stop":1737570609082,"duration":8697}},{"uid":"a29a903e754fc94e","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568967477,"stop":1737568967477,"duration":0}},{"uid":"8418c3314df6b953","status":"passed","time":{"start":1734034855790,"stop":1734034861092,"duration":5302}},{"uid":"c53b174c7184e8d1","status":"passed","time":{"start":1733937568833,"stop":1733937575595,"duration":6762}},{"uid":"303e399fd77bf1cc","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@class=\"onboarding_container__mDp2D\"]')","time":{"start":1733866107121,"stop":1733866107121,"duration":0}},{"uid":"a5f3d2e237b63114","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')","time":{"start":1733860554361,"stop":1733860554361,"duration":0}},{"uid":"92ee359572d99e98","status":"passed","time":{"start":1733857138917,"stop":1733857142759,"duration":3842}},{"uid":"e3ef48c9115d2137","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')","time":{"start":1733772024421,"stop":1733772024421,"duration":0}},{"uid":"3afd4059a120e583","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найдены элементы с локатором ('xpath', '//div[@class=\"user-project-list_count__TcOiJ\"]')","time":{"start":1733766505502,"stop":1733766535668,"duration":30166}}]},"tags":["regression","profile"]},"source":"59a223c0c6ec5622.json","parameterValues":[]}